### PR TITLE
Telephony Guide Round 3

### PR DESCRIPTION
**Pull request by isaacdurham-okta**
## Description:
- **What's changed?** Slight rollback on previous changes, URL adjustment, and note removal.
- **Is this PR related to a Monolith release?** Nope

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/acf27384d876db0f0156a22f819ea136a95ed284/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c3a434ba-14aa-443e-9002-dc68d6669d27) | 92  | 72  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.cde33e5c-e19e-4986-8991-e77c9ab46048?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/6c68877c45c2fbdcf870e56f0b2839f9f5f5054d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/819a45d9-3d47-4a82-a20d-89a445c389e3) | 92  | 75  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.f9b04fe0-f28a-4763-b5ae-300e907b14a9?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 1]
```diff
@@ -151,9 +151,7 @@ To tell Okta that the SMS or voice call message was successfully sent, return a
 
 ## Preview and test
 
-The external service example is now ready with code to receive and respond to an Okta call. The Okta org is set up to call the external service using a telephony inline hook. In your Okta org, preview the JSON-formatted request and response directly from the Admin Console. You can also test the code directly in your org. 
-
-The following steps use Okta Workflows, but you can also use the Inline Hooks Management API. See [Telephony inline hook reference](https://developer.okta.com/docs/reference/telephony-hook).
+The external service example is now ready with code to receive and respond to an Okta call. The Okta org is set up to call the external service using a telephony inline hook. In your Okta org, preview the JSON-formatted request and response directly from the Admin Console. You can also test the code directly in your org.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:154]:
```suggestion
The external service example is now ready with code to receive and respond to an Okta call. The Okta org is set up to call the external service using a telephony inline hook. In your Okta org, preview the JSON-formatted request and response directly from the Admin Console.
```

**Review by brianduffield-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/cd365f667f8d161c8dbe409518b84b5c77f02471/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/08f2d1f6-17c8-4743-accd-1b760dd3deda) | 92  | 75  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.a80e7e95-f887-4b0a-8093-a19a50ae5979?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**